modules/rpm-ostree/README.md

```diff
@@ -4,12 +4,24 @@ The [`rpm-ostree`](https://coreos.github.io/rpm-ostree/) module offers pseudo-de
 
 The module first downloads the repository files from repositories declared under `repos:` into `/etc/yum.repos.d/`. The magic string `%OS_VERSION%` is substituted with the current VERSION_ID (major Fedora version), which can be used, for example, for pulling correct versions of repositories from [Fedora's Copr](https://copr.fedorainfracloud.org/).
 
+You can also have local repos inserted in this file location (make the directory if it doesn't exist):  
+```
+files/rpm-ostree/my-repository.repo
+files/rpm-ostree/my-image/my-2nd-repository.repo
+```
+
 If you use a repo that requires adding custom keys (eg. Brave Browser), you can import the keys by declaring the key URLs under `keys:`. The magic string acts the same as it does in `repos`.
 
 Then the module installs the packages declared under `install:` using `rpm-ostree install`, it removes the packages declared under `remove:` using `rpm-ostree override remove`. If there are packages declared under both `install:` and `remove:` a hybrid command `rpm-ostree remove <packages> --install <packages>` is used, which should allow you to switch required packages for other ones.
 
 Installing RPM packages directly from a `http(s)` url that points to the RPM file is also supported, you can just put the URLs under `install:` and they'll be installed along with the other packages. The magic string `%OS_VERSION%` is substituted with the current VERSION_ID (major Fedora version) like with the `repos:` property.
 
+Installing local RPM packages from the file location is supported. You can insert those RPMs in this file location (make the directory if it doesn't exist):  
+```
+files/rpm-ostree/my-package.rpm
+files/rpm-ostree/my-image/somepackage.rpm
+```
+
```

Comment:
````suggestion
If an RPM is not available in a repository or as an URL, you can also install it directly from a file in your git repository. For example:
```yml
install:
   - weird-package.rpm # tries to install files/rpm-ostree/weird-package.rpm
```
````